cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunExecutableTask.java

```diff
@@ -53,7 +54,7 @@
 public class RunExecutableTask implements Task {
 
     private final String[] args;
-    private Path executablePath;
+    private Path executableJarPath;
```

Comment:
this is module jar i think?

Reply:
Yeah this is the module executable jar file path. Do you have any alternative suggestions for the var name?